swarm/pss/pss.go

```diff
@@ -150,14 +186,12 @@ func (self *Pss) APIs() []rpc.API {
 			Service:   NewAPI(self),
 			Public:    true,
 		},
-	}
-	if self.debug {
-		apis = append(apis, rpc.API{
-			Namespace: "pss",
+		rpc.API{
+			Namespace: "psstest",
 			Version:   "0.1",
 			Service:   NewAPITest(self),
 			Public:    true,
```

Comment:
there's a way to make it "private" and still use it? We need the API still in the development phase.